src/Orckestra.Composer.Search/Providers/BaseSearchCriteriaProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Orckestra.Composer.Parameters;
+using Orckestra.Composer.Providers;
+using Orckestra.Composer.Search.RequestConstants;
+using Orckestra.Composer.Services;
+using System;
+using System.Threading.Tasks;
+
+namespace Orckestra.Composer.Search.Providers
+{
+    public class BaseSearchCriteriaProvider : IBaseSearchCriteriaProvider
+    {
+        protected IWebsiteContext WebsiteContext { get; private set; }
+        protected IInventoryLocationProvider InventoryLocationProvider { get; private set; }
+        protected IComposerContext ComposerContext { get; private set; }
+        protected IFulfillmentContext FulfillmentContext { get; private set; }
+
+        public BaseSearchCriteriaProvider(IWebsiteContext websiteContext, IInventoryLocationProvider inventoryLocationProvider, IComposerContext composerContext, IFulfillmentContext fulfillmentContext)
+        {
+            WebsiteContext = websiteContext ?? throw new ArgumentNullException(nameof(websiteContext)); ;
+            InventoryLocationProvider = inventoryLocationProvider ?? throw new ArgumentNullException(nameof(inventoryLocationProvider));
+            ComposerContext = composerContext ?? throw new ArgumentNullException(nameof(composerContext));
+            FulfillmentContext = fulfillmentContext ?? throw new ArgumentNullException(nameof(fulfillmentContext));
+        }
+
+        public async Task<SearchCriteria> GetSearchCriteria(string searchTerms, int limit, int offset, string baseURL, bool includeFacets)
+        {
+            return new SearchCriteria
+            {
+                Keywords = searchTerms,
+                NumberOfItemsPerPage = limit,
+                IncludeFacets = includeFacets,
+                StartingIndex = offset,
+                Page = 1,
+                BaseUrl = baseURL,
+                Scope = ComposerContext.Scope,
+                CultureInfo = ComposerContext.CultureInfo,
+                InventoryLocationIds = await InventoryLocationProvider.GetInventoryLocationIdsForSearchAsync().ConfigureAwait(false),
+                AvailabilityDate = FulfillmentContext.AvailabilityAndPriceDate,
+                AutoCorrect = SearchConfiguration.AutoCorrectSearchTerms,
```

Comment:
Probably we need add configuration for default SortBy and SortDirection 
 SortBy = "score"
SortDirection  ="desc"
so search and autocomplete by default have same
Then search can override